src/core/scene/a-scene.js

```diff
@@ -450,10 +450,23 @@ module.exports.AScene = registerElement('a-scene', {
       value: function () {
         var camera = this.camera;
         var canvas = this.canvas;
-        var embedded = this.getAttribute('embedded') && !this.is('vr-mode');
+        var embedded;
         var size;
         var vrDevice;
         var isVRPresenting;
+        // Workaround for a Webkit bug (https://bugs.webkit.org/show_bug.cgi?id=170595)
+        // where the window does not contain the correct viewport size
+        // after an orientation change. The window size is correct if the operation
+        // is postponed a few miliseconds.
+        // The if satement below can be removed when the bug is fixed.
+        if (this.iOS && !this.delayedResize) {
```

Comment:
Wouldn't it be simpler to go to the first `this.resize()` call and do the timeout check there versus binding + keeping state?

```
if (isIOS) {
  setTimeout(this.resize.bind(this), 200);
} else {
  this.resize();
}
```

Reply:
That will work too

Reply:
changed